frontend/src/components/user/ProfileCard.tsx

```diff
@@ -28,7 +28,7 @@ const ProfileCard: React.FC<Props> = (props): JSX.Element => {
         <div className="flex flex-col items-center pb-3">
             <div className="inline-flex rounded-full shadow-lg">
               <Avatar 
-                url={`${user ? user?.avatar_url : 'https://i.stack.imgur.com/l60Hf.png'}`}
+                url={`${user?.avatar_url === null ? 'https://i.stack.imgur.com/l60Hf.png' : user?.avatar_url}`}
```

Comment:
put default avatar `url` into a constant variable and remove redundant `?` for the else statement